CHANGELOG.rst

```diff
@@ -19,6 +19,8 @@ Removed
 
 Fixed
 =====
+Fixed some links using sphinx documentation.
+Fixed all sphinx warnings during builds.
```

Comment:
I would write something more high-level in the changelog like:
- Improved documentation: fixed links and build warnings.